zetaclient/chains/bitcoin/observer/inbound.go

```diff
@@ -333,26 +351,31 @@ func FilterAndParseIncomingTx(
 	tssAddress string,
 	logger zerolog.Logger,
 	netParams *chaincfg.Params,
-	depositorFee float64,
 ) ([]*BTCInboundEvent, error) {
-	inbounds := make([]*BTCInboundEvent, 0)
+	events := make([]*BTCInboundEvent, 0)
```

Comment:
```suggestion
	events := make([]*BTCInboundEvent, 0,len(txs))
```

Reply:
The number of `events` will always be  `< len(txs)`, mostly `0`. So we append them one by one if found.

Reply:
Yeah, it's a minor nitpick, but I think the array is reallocated to twice the size every time it gets full; starting with a max capacity of len(txt) would save some allocations.